README.md

```diff
@@ -214,7 +214,14 @@ const bot = new Telegraf(token);
 bot.on("text", ctx => ctx.reply("Hello"));
 
 // Start webhook via launch method (preferred)
-bot.launch({ webhook: { domain: webhookDomain, port: port } });
+bot.launch({ 
+  webhook: {
+    domain: webhookDomain, // https://example.com
+    port: port, // 443
+    hookPath: webhookPath, // Application path on server
```

Comment:
"If not passed, Telegraf will generate a secure path for you."

Reply:
Done